src/events/shard/shardReady.ts

```diff
@@ -0,0 +1,10 @@
+import { ShardEvent } from '@lib/structures/ShardEvent';
+import { green } from 'colorette';
+
+export default class extends ShardEvent {
+	protected readonly title = green('Ready');
+
+	public run(id: number, unavailableGuilds: Set<string> | null) {
+		this.client.console.error(`${this.header(id)}: ${unavailableGuilds?.size ?? 'Unknown'} Guilds`);
```

Comment:
```suggestion
		this.client.console.error(`${this.header(id)}: ${unavailableGuilds?.size ?? 'Unknown or no unavailable'} guilds`);
```

Reply:
Resolved all requested changes in 027243f6.